projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/contrast/colors.scss

```diff
@@ -41,4 +41,15 @@ $color-primary-light: $studio-gray-60; // Calypso --color-primary-light
 $color-surface: $studio-white; // Calypso --color-surface
 $color-neutral-100-rgb: $studio-gray-100-rgb; // Calypso --color-neutral-100-rgb
 
+// Sidebar Nudges
+$menu-nudge-background: $studio-gray-80;
+$menu-nudge-text-color: $studio-white;
+$menu-nudge-cta-background: $highlight-color;
+$menu-nudge-cta-color: $studio-white;
+$menu-nudge-cta-background-hover: $studio-blue-60;
+
+.folded #adminmenu .toplevel_page_site-notices .wp-menu-image {
+	background-color: $studio-gray-80 !important;
+}
```

Comment:
This breaks the current pattern of only defining variables in the color schemes files. Can we maybe move it to `_overrides.css` which already contains an exception for the contrast scheme?

https://github.com/Automattic/jetpack/blob/3c7a142a36aa541c71917ee797a91e370c32de17/projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_overrides.scss#L133-L136

Reply:
Never mind, just realized there are other color schemes files with CSS rules likes `fresh` or `light`.

Reply:
Moved here e8b68bc